woocommerce/google-listings-and-ads
# Setup MC: Supported Countries, Step 2 Target Audience, and Step 3 Country Selection in Shipping Rate

### Changes proposed in this Pull Request:

Closes #143 .

This PR makes uses of the API from https://github.com/woocommerce/google-listings-and-ads/pull/142. 

This PR contains the following: 

- Loads the supported countries from API in Step 2.
- Load and save the target audience data automatically in Step 2.
- Loads the list of countries in Shipping Rate and Shipping Time sections in Step 3 based on your target audience countries in Step 2.

### Screenshots:

Watch my demo video for the Step 2 page (turn on your speaker), this shows calls to supported countries API and target audience API: https://d.pr/v/APvmML

### Detailed test instructions:

1. Go to Step 2 in https://gla1.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fsetup-mc
2. Modify the form by changing the radio option and the selected countries. The page should auto save your form values.
3. Reload the page. The page should auto load the previously saved values.
4. Take note of your selected location option and countries.
5. Go to Step 3.
6. Test the shipping rate section by clicking on the edit button and the add button. The country select dropdown should display a list of countries that corresponds to your target audience countries.
